### ISSUE TITLE
Missing HKCU registry value

### ISSUE THREAD

**Issue by RobsonMi** [EvenAR/node-simconnect#46]
When starting 3.0.1-3.0.4 version I am getting this error:
![Zrzut ekranu 2022-09-28 233517](https://user-images.githubusercontent.com/15092964/192893457-341590a1-e05a-4f89-80ed-241dbafadbe4.png)

It still connects to sim (MSFS Steam in this case) but why would it need to check SimConnect port in registry?

**Comment by BarryCarlyon** [comment]
> why would it need to check SimConnect port in registry?

In case it's not on the default port. So it's trying to determine automatically the port.

**Comment by RobsonMi** [comment]
Any way I can catch the error thrown in Utils.ts at line 21?
`throw Error(`Failed to read registry value ${FS_KEY} (${err})`);`

**Comment by BarryCarlyon** [comment]
In lieu of a "module fix to make it graceful catch"

```
process.on('uncaughtException', err => {
    console.error('uncaughtException', err);
});
```

might catch it, if the log is saying it's a uncaught exception

**Comment by RobsonMi** [comment]
That's not graceful at all. No idea what's the best solution here but surely not that error message ;) Maybe simple console.error with info that default will be used?

**Comment by EvenAR** [comment]
This should have been fixed now in v3.0.5 (thanks for the PR @RobsonMi)
